client/signup/steps/theme-selection/index.jsx

```diff
@@ -38,16 +38,19 @@ module.exports = React.createClass( {
 	},
 
 	pickTheme( theme ) {
-		const themeSlug = theme.id;
+		const repoSlug = `${ theme.repo }/${ theme.slug }`;
 
-		analytics.tracks.recordEvent( 'calypso_signup_theme_select', { theme: themeSlug, headstart: true } );
+		analytics.tracks.recordEvent( 'calypso_signup_theme_select', {
+			theme: repoSlug,
+			headstart: true
+		} );
 
 		SignupActions.submitSignupStep( {
 			stepName: this.props.stepName,
 			processingMessage: this.translate( 'Adding your theme' ),
-			themeSlug
+			repoSlug
```

Comment:
Ah, no; that `themeSlug` in your link comes from [here](https://github.com/Automattic/wp-calypso/blob/master/client/signup/steps/domains/index.jsx#L134), and that theme is [based on query args](https://github.com/Automattic/wp-calypso/blob/master/client/signup/steps/domains/index.jsx#L90) that may be set for the `with-theme` flow. At the line you mentioned, `themeSlug`, `themeSlugWithRepo`, and `themeItem` are all `undefined` if in a flow other than `with-theme`.